src/setfit/onnx.py

```diff
@@ -0,0 +1,281 @@
+import warnings
+from typing import Callable, Optional, Union
+
+import numpy as np
+import onnx
+import torch
+from sentence_transformers import SentenceTransformer, models
+from sklearn.linear_model import LogisticRegression
+from torch import nn
+from transformers.modeling_utils import PreTrainedModel
+
+
+def mean_pooling(token_embeddings: torch.Tensor, attention_mask: torch.Tensor) -> torch.Tensor:
+    """Perform attention aware mean pooling.
+
+    This method takes in embeddings of shape (batch, sequence, embedding_size) and performs average
+    pooling across the sequence dimension to yield embeddings of size (batch, embedding_size).
+
+    From:
+    https://huggingface.co/sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2#usage-huggingface-transformers
```

Comment:
Let's cite the source in case the model repo changes :)
```suggestion
    https://github.com/UKPLab/sentence-transformers/blob/0b5ef4be93d2b21de3a918a084b48aab6ba48595/sentence_transformers/model_card_templates.py#L134
```